src/lib/viewers/controls/fullscreen/FullscreenToggle.tsx

```diff
@@ -6,7 +6,7 @@ import './FullscreenToggle.scss';
 
 export type Props = {
     onFullscreenToggle: (isFullscreen: boolean, toggleFullscreenEl: EventTarget | null) => void;
-    onKeyDown?: (event: React.KeyboardEvent<HTMLButtonElement>) => void;
+    onKeyDown?: (keyboardEvent: React.KeyboardEvent<HTMLButtonElement>) => void;
```

Comment:
Can we revert this change now that the `no-unrestricted-globals` issue is fixed?